presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestMaterializedViewQueryOptimizer.java

```diff
@@ -0,0 +1,304 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.analyzer;
+
+import com.facebook.presto.sql.parser.SqlParser;
+import com.facebook.presto.sql.tree.QualifiedName;
+import com.facebook.presto.sql.tree.Query;
+import com.facebook.presto.sql.tree.Table;
+import org.testng.annotations.Test;
+
+import static java.lang.String.format;
+import static org.testng.Assert.assertEquals;
+
+public class TestMaterializedViewQueryOptimizer
+{
+    private static SqlParser sqlParser = new SqlParser();
+
+    @Test
+    public void testWithSimpleQuery()
+    {
+        String baseTable = "base_table";
```

Comment:
@jainxrohit If we are still using this file for Join test cases, it would be better to leave it like this (in order to add more table name). If not, we can move it to constant.

Reply:
sure not a strong preference. Since you are using same table name in all tests, you might end with same table1, table2 for joins as well. We can leave it as it is.